Products/ZenUI3/browser/configure.zcml

```diff
@@ -398,7 +398,8 @@
 
 <browser:viewlet
     name="zenoss_data"
-    manager=".interfaces.IMainSnippetManager"
+    weight="3"
+    manager=".interfaces.IJavaScriptSrcManager"
```

Comment:
I had to change manager because `zenoss_data` in no longer a JavaScript snippet, but a whole `<script>` element, as you spotted in your other comment. Here I'll try explain why I made such a change.

In case you don't remember, a difference between managers is that `IMainSnippetManager` collects JS snippets and packs it into one `<script>` node, whereas `IJavaScriptSrcManager` is expecting a `<script>` node. Code grouped together by `IMainSnippetManager` is included after all `<script>` tags provided by `IJavaScriptSrcManager`.

The problem is that `zenoss-data` was originally included after `visualization-library` (see Products/ZenUI3/browser/javascript.zcml file) but now `visualization-library` depends on `zenoss-data`.

I couldn't move whole node which was rendered by `IMainSnippetManager` before `visualization-library` because it has its own dependencies (which are loaded by `IJavaScriptSrcManager`...). Thus I had to extract `zenoss-data` from `IJavaScriptSrcManager` and transform it into independent node, wrapping it in it's own `<script>` tag.